docs/deployment/restorePDMultiRegionSeedFailure.md

```diff
@@ -0,0 +1,164 @@
+# Restoring a Multi Region PingDirectory Deployment After Seed Cluster Failure
+
+The PingDirectory hook scripts rely on the seed server of the seed cluster being available when running in a multi-region environment. This leads to the question of what can be done if the seed region's servers, along with their persistent volumes are lost. This page describes manual steps that can be taken to restore the PingDirectory topology in this case.
```

Comment:
environment.  This **dependency** leads to ...

... along with their persistent volumes, are lost.